x-pack/plugins/ml/public/jobs/jobs_list_new/components/job_filter_bar/job_filter_bar.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+import PropTypes from 'prop-types';
+import React, {
+  Component
+} from 'react';
+
+import { ml } from 'plugins/ml/services/ml_api_service';
+import { JobGroup } from '../job_group';
+
+import './styles/main.less';
+
+import {
+  EuiSearchBar,
+} from '@elastic/eui';
+
+const loadGroups = () => {
+  return ml.jobs.groups()
+    .then((groups) => {
+      return groups.map(g => ({
```

Comment:
The style guide recommends using parentheses when the function body includes braces